### PR TITLE
refactor: Update xapi_events_all_parsed and supporting view

### PR DESCRIPTION
**Pull request by bmtcril**
- Extract org and object id into the top level of xapi_events_all_parsed.
- Use the String type instead of JSON for the event JSON data, since the JSON type is so poorly supported.
- Add time_last_dumped to event_sink.relationships PK to help speed up queries on that data

### CODE REVIEW

**Comment by bmtcril** [comment]
@Ian2012 @mariajgrimaldi @soryrawyer I can't ping you all for reviews directly, but would appreciate your eyes on this. It should tidy up some longstanding issues with this table, but once merged will require you to drop the `xapi_events_all_parsed` table and `xapi_events_all_parsed_mv` view so they can be recreated. Hopefully that will be the last time we need to do this, but I kinda doubt it.

File: tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh [hunk 1]
```diff
@@ -31,11 +40,12 @@ PRIMARY KEY (emission_time, event_id);
...
     event_id UUID NOT NULL,
     verb_id String NOT NULL,
-    actor_id UUID NOT NULL,
+    actor_id String NOT NULL,
+    object_id String NOT NULL,
     org String NOT NULL,
     course_id String NOT NULL,
     emission_time DateTime64(6) NOT NULL,
-    event JSON NOT NULL
+    event_str String NOT NULL
```
Comment (pomegranited) [inline tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh:48]: I don't understand how  the JSON data type is poorly supported in Clickhouse? I've been loving how I can dig into it with the SQLExplorer in Superset, and it makes it very easy to dig out event specifics that aren't in their own fields (like whether a problem was answered successfully or not).

Comment (pomegranited) [reply tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh:48]: Ok, sure.. that's still a shame! I had hoped we could avoid many tedious schema changes whenever we want to chart new xAPI event types! But oh well..

I guess anyone who's in a rush can still use the https://clickhouse.com/docs/en/sql-reference/functions/json-functions

**Review by pomegranited — APPROVED** [review]
:+1: LGTM

- [x] I tested this by spinning up a fresh OARS stack, running ` tutor local do load-xapi-test-data`, and checking the xapi tables are populated as expected. Also ensured that new events are still percolating through from the LMS.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A

**Pull request merged**